app/src/main/java/org/smartregister/addo/intent/AddoPncCloseIntent.java

```diff
@@ -0,0 +1,42 @@
+package org.smartregister.addo.intent;
+
+import android.app.IntentService;
+import android.content.Intent;
+
+import androidx.annotation.Nullable;
+
+import org.smartregister.chw.pnc.PncLibrary;
+import org.smartregister.chw.pnc.repository.PncCloseDateRepository;
+import org.smartregister.chw.pnc.util.PncUtil;
+
+import timber.log.Timber;
+
+public class AddoPncCloseIntent extends IntentService {
+
+    private PncCloseDateRepository repository;
+    private static final String TAG = "AddoPncCloseIntent";
+
+    public AddoPncCloseIntent() { super(TAG); }
+
+    @Override
+    public int onStartCommand(@Nullable Intent intent, int flags, int startId) {
+        repository = PncLibrary.getInstance().getPncCloseDateRepository();
+        return super.onStartCommand(intent, flags, startId);
+    }
+
+    public int getNumberOfPncDays() {
```

Comment:
is it day or years?

Reply:
@Andati - PNC is Days. It is 43 days. Like in CHV

Reply:
cool